### PR TITLE
feat: hebrew cloudevents spec

### LINKED ISSUE #1042: translation: cloudevents spec to hebrew
I want to translate the CloudEvents spec to Hebrew. 

The issues I think I will encounter is RLT support from github

Never done this before but I think this will be cool experience

### PR DESCRIPTION
**Pull request by sasha-tkachev**
Fixes #1042

## Proposed Changes

- Translate `spec.md` under cloudevents to hebrew

**Release Note**

```release-note
Translate `spec.md` under cloudevents to hebrew
```

### CODE REVIEW

**Comment by duglin** [comment]
@sasha-tkachev let me know when you think this one is ready for review

**Comment by duglin** [comment]
Also, when ready, can you squash it down to one commit?

File: cloudevents/languages/he/README.md [hunk 1]
```diff
@@ -0,0 +1 @@
+מסמך זה תרם תורגם. בבקשה השתמשו [בגרסה האנגלית של המסמך](../../README.md) לבינתיים.
```
Comment (duglin) [inline cloudevents/languages/he/README.md:1]: missing "not" ?

**Comment by duglin** [comment]
LGTM

**Comment by duglin** [comment]
@sasha-tkachev the href checker isn't happy

**Comment by duglin** [comment]
Approved on the 8/18 call and will merge when href fix is done

**Comment by sasha-tkachev** [comment]
@duglin seems that the href check fails on hebrew section names, I will need to probably edit the verify docs script

**Comment by duglin** [comment]
@JieDing I just noticed that the links in the TOC on https://github.com/cloudevents/spec/blob/main/cloudevents/languages/zh-CN/spec.md don't work and I think it's due to the chinese characters (after the `/` in the title) missing from the link.  
@sasha-tkachev let me see what's going on in for you...

**Comment by JieDing** [comment]
> @JieDing I just noticed that the links in the TOC on https://github.com/cloudevents/spec/blob/main/cloudevents/languages/zh-CN/spec.md don't work and I think it's due to the chinese characters (after the `/` in the title) missing from the link. Don't know why the link checker didn't catch it - but perhaps it's related to @sasha-tkachev's issue. @sasha-tkachev let me see what's going on in for you...

@duglin That's pretty weird. I just found same problems happened on TOCs on different files. I'll fix it asap. Also don't know why the link checker didn't catch it.

**Comment by duglin** [comment]
@sasha-tkachev testing https://github.com/cloudevents/spec/pull/1057 to see if it'll fix the issue you found. However, it did uncover some bad links in your spec.md - you used uppercase letters in some of your links.

**Comment by duglin** [comment]
@JieDing pick up my changes in #1057 - I think that should fix the href checker and it might fix a lot of the TOC issues

**Comment by duglin** [comment]
@sasha-tkachev if you rebase on #1057 you might be able to see the remaining link issues.

**Comment by duglin** [comment]
I think these are the ones that need to be lower-cased:
```
./cloudevents/languages/he/spec.md: Can't find section '#יצרן---Producer' in ./cloudevents/languages/he/spec.md
./cloudevents/languages/he/spec.md: Can't find section '#צרכן---Consumer' in ./cloudevents/languages/he/spec.md
./cloudevents/languages/he/spec.md: Can't find section '#מתווך---Intermediary' in ./cloudevents/languages/he/spec.md
./cloudevents/languages/he/spec.md: Can't find section '#יצרן---Producer' in ./cloudevents/languages/he/spec.md
./cloudevents/languages/he/spec.md: Can't find section '#צרכן---Consumer' in ./cloudevents/languages/he/spec.md
./cloudevents/languages/he/spec.md: Can't find section '#מאפייני-הקשרContext-Attributes' in ./cloudevents/languages/he/spec.md
./cloudevents/languages/he/spec.md: Can't find section '#מאפייני-הקשרContext-Attributes' in ./cloudevents/languages/he/spec.md
./cloudevents/languages/he/spec.md: Can't find section '#מיפוי-פרוטוקול---Protocol-Binding' in ./cloudevents/languages/he/spec.md
./cloudevents/languages/he/spec.md: Can't find section '#פורמט-האירועEvent-Format' in ./cloudevents/languages/he/spec.md
./cloudevents/languages/he/spec.md: Can't find section '#הרחבת-מאפייני-הקשר---Extension-Context-Attributes' in ./cloudevents/languages/he/spec.md
./cloudevents/languages/he/spec.md: Can't find section '#מוסכמות-סימון---Notational-Conventions' in ./cloudevents/languages/he/spec.md
./cloudevents/languages/he/spec.md: Can't find section '#מוסכמות-סימון---Notational-Conventions' in ./cloudevents/languages/he/spec.md
./cloudevents/languages/he/spec.md: Can't find section '#סוגי-טיפוסים---Type-System' in ./cloudevents/languages/he/spec.md
./cloudevents/languages/he/spec.md: Can't find section '#נתונים---Data' in ./cloudevents/languages/he/spec.md
```

**Comment by sasha-tkachev** [comment]
@duglin Thank you very much! 
I was just about to start writing the script myself

**Comment by duglin** [comment]
links are fixed! merging

**Pull request merged**